lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
+
+
+class BaseCommand(object):
+    member = None
+    argument_mappings = None
+    return_mapping = None
```

Comment:
I had completely forgotten that `argument_mappings` could be arbitrary transformation functions.

Not sure I have better terms. In a way, these are pre- and post-processing... filters for lack of a better work.. Probably best to leave it as-is in that case.